account_asset_analytic/models/account_invoice_line.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api
+
+
+class AccountInvoceLine(models.Model):
+    _inherit = 'account.invoice.line'
+
+    @api.model
+    def asset_create(self, lines):
+        res = super(AccountInvoceLine, self).asset_create(lines)
```

Comment:
Put a comment indicating that this propagates the analytic account from the invoice.